packages/react/src/themes/build-theme.test.ts

```diff
@@ -0,0 +1,118 @@
+import { devConsole } from '../utils/dev-console';
+import { buildTheme } from './build-theme';
+import { ThemeCustomization } from './theme';
+import { defaultRefTokens, defaultAliasTokens } from './tokens';
+
+const customization : ThemeCustomization = {
+    ref: {
+        'color-brand-05': 'red',
+        'color-brand-20': 'green',
+        'color-neutral-02': 'blue',
+    },
+    alias: {
+        'default-text-color': 'color-brand-05',
+        'alternate-text-color': 'color-brand-20',
+    },
+    component: {
+        'button-primary-background-color': 'color-neutral-02',
+        'button-primary-inverted-background-color': 'color-brand-20',
+        'focus-inside-border-color': 'alternate-text-color',
+    },
+};
+
+const expectedTheme = {
+    ref: {
+        ...defaultRefTokens,
+        'color-brand-05': 'red',
+        'color-brand-20': 'green',
+        'color-neutral-02': 'blue',
+    },
+    alias: {
+        ...defaultAliasTokens,
+        'default-text-color': 'color-brand-05',
+        'alternate-text-color': 'color-brand-20',
+    },
+    component: {
+        'button-primary-background-color': 'blue',
+        'button-primary-inverted-background-color': 'green',
+        'button-focus-border': 'red',
+        'button-primary-border-color': '#006296',
+    },
+};
+
+describe('buildTheme', () => {
+    it('should build the defaultRefTokens theme with the customization provided', () => {
+        const builtTheme = buildTheme(customization);
+
+        expect(builtTheme.ref).toEqual(expectedTheme.ref);
+    });
+    it('should build the defaultAliasTokens theme with the customization provided', () => {
```

Comment:
C'est un détail, mais je mettrais une ligne vide entre chacun des tests.
```suggestion
    });

    it('should build the defaultAliasTokens theme with the customization provided', () => {
```